Travis Fails

Travis fails by my last merges, however I am pretty sure these merges are not the reason for this. If you look at the PRs list, you will notice our latest pull requests are always failed with Travis. This log may be meaningful:

```
FAILURE: Build failed with an exception.
* What went wrong:
A problem occurred configuring project ':app'.
You have not accepted the license agreements of the following SDK components:
[Android SDK Platform 26].
Before building your project, you need to accept the license agreements and complete the installation of the missing components using the Android Studio SDK Manager.
```
Do you have any idea about how to accept license agreements?

--- a/.travis.yml
+++ b/.travis.yml
@@ -24,7 +24,10 @@ android:
     - extra-android-m2repository
     - ${ANDROID_TARGET}
     - android-25
+    - android-26
     - sys-img-${ANDROID_ABI}-${ANDROID_TARGET}
+  licenses:
+    - 'android-sdk-license-.+'
 
 before_script:
   - echo no | android create avd --force -n test -t $ANDROID_TARGET --abi $ANDROID_ABI

